src/python/WMCore/WMSpec/WMTask.py

```diff
@@ -897,6 +897,7 @@ def setSubscriptionInformation(self, custodialSites = None, nonCustodialSites =
                                    custodialSubType = "Replica", nonCustodialSubType = "Replica",
                                    custodialGroup = "DataOps", nonCustodialGroup = "DataOps",
                                    priority = "Low", primaryDataset = None,
+                                   useSkim = False, isSkim = False,
```

Comment:
This two parameter need to be moved to at the end. Following uses positional arguments
https://github.com/dmwm/WMCore/blob/master/src/python/WMCore/WMSpec/WMWorkload.py#L1224
https://github.com/dmwm/WMCore/blob/master/src/python/WMCore/WMSpec/WMWorkload.py#L1230